Makefile

```diff
@@ -134,9 +134,13 @@ build-deploy-certmanager:
 .PHONY: build-deploy-dev-environment
 build-deploy-dev-environment: build-deploy-certmanager build-deploy-operator build-deploy-metrics-operator build-deploy-scheduler
 
-
 include docs/Makefile
 
+.PHONY: mkdocs-deploy
```

Comment:
why not go with the same make commands that we had in the old docs makefile?
`deploy` is kind of ambiguous because `mkdocs` actually has a deploy cmd which we don't need at all

Reply:
I think you cannot add them to the root Makefile afterwards (you will have the same commands defined 2x)